xonsh/completer.py

```diff
@@ -7,12 +7,12 @@
 import subprocess
 import sys
 
+from builtins import __xonsh_env__ as ENV
```

Comment:
I converted these to a global because this was a recurring pattern. I'm running off of this version and it seems to be working without errors and as expected.

If you feel strongly tho, I can do it as you suggested (and did before).

A mid-way solution would be to have a global `GET_ENV` method as follows:

```py
GET_ENV = lambda name, default=None: builtins.__xonsh_env__.get(name, default)
```